File: JAVA/Petshop.java

public class Petshop {
    private int id;
    private String namaProduk;
    private int hargaProduk;
    private int stokProduk;

    public Petshop() {
        this.id = 1;
        this.namaProduk = "";
        this.hargaProduk = 0;
        this.stokProduk = 0;
    }

    public Petshop(int id, String namaProduk, int hargaProduk, int stokProduk) {
        this.id = id;
        this.namaProduk = namaProduk;
        this.hargaProduk = hargaProduk;
        this.stokProduk = stokProduk;
    }

    public void setId(int id) {
        this.id = id;
    }

    public void setNamaProduk(String namaProduk) {
        this.namaProduk = namaProduk;
    }

    public void setHargaProduk(int hargaProduk) {
        this.hargaProduk = hargaProduk;
    }

    public void setStokProduk(int stokProduk) {
        this.stokProduk = stokProduk;
    }

    public int getId() {
        return id;
    }

    public String getNamaProduk() {
        return namaProduk;
    }

    public int getHargaProduk() {
        return hargaProduk;
    }

    public int getStokProduk() {
        return stokProduk;
    }
}

File: CPP/Petshop.cpp
#include <iostream>
#include <string>
#include <list>

using namespace std;

class Petshop {
    private:
        int id;
        string namaProduk;
        int hargaProduk;
        int stokProduk;

    public:
        Petshop(){
            id = 1;
            namaProduk = "";
            hargaProduk = 0;
            stokProduk = 0;
        }

        Petshop(int id, string namaProduk, int hargaProduk, int stokProduk){
            this->id = id;
            this->namaProduk = namaProduk;
            this->hargaProduk = hargaProduk;
            this->stokProduk = stokProduk;
        }

        void setId(int id){
            this->id = id;
        }

        void setNamaProduk(string namaProduk){
            this->namaProduk = namaProduk;
        }

        void setHargaProduk(int hargaProduk){
            this->hargaProduk = hargaProduk;
        }

        void setStokProduk(int stokProduk){
            this->stokProduk = stokProduk;
        }

        int getId(){
            return id;
        }

        string getNamaProduk(){
            return namaProduk;
        }

        int getHargaProduk(){
            return hargaProduk;
        }

        int getStokProduk(){
            return stokProduk;
        }

        ~Petshop(){
        }

};